src/services/dashboards/resource.js

```diff
@@ -330,6 +331,438 @@ const switchToTopicCentric = (input) => {
+      tt."totalCount"
+    FROM ${tblNames.createdTopicsTempTableName} d
+    JOIN series s
+      ON 1=1
+    JOIN totals tt
+      ON d.name = tt.name
+    LEFT JOIN topicsperdate t
+      ON d.name = t.name AND to_char(s."date", 'Mon-YY') = t."rollUpDate"
+    ORDER BY 1, 3 ASC;`;
+  return sequelize.query(
+    topicUseSql,
+    {
+      type: QueryTypes.SELECT,
+      transaction,
+    },
+  );
+}
+
+function getOverview(tblNames, totalReportCount, transaction) {
+  // - Number of Participants -
+  const numberOfParticipants = sequelize.query(`
+  WITH ar_participants AS (
+    SELECT
+    id,
+    "numberOfParticipants"
+    FROM ${tblNames.createdFlatResourceTempTableName} f
+    GROUP BY id, "numberOfParticipants"
+  )
+  SELECT
+         SUM("numberOfParticipants") AS participants
+  FROM ar_participants;
+  `, {
+    type: QueryTypes.SELECT,
+    transaction,
+  });
+
+  const numberOfRecipSql = `
+  WITH ars AS (
+    SELECT
+    DISTINCT id
+    FROM ${tblNames.createdFlatResourceTempTableName} f
+  ), recipients AS (
+    SELECT
+      DISTINCT r.id
+    FROM ars ar
+    JOIN "ActivityRecipients" arr
+      ON ar.id = arr."activityReportId"
+    JOIN "Grants" g
+      ON arr."grantId" = g.id AND g."status" = 'Active'
+     JOIN "Recipients" r
+      ON g."recipientId" = r.id
+  )
+  SELECT
+         count(r.id) AS recipients
+  FROM recipients r;
+  `;
+
+  // - Number of Recipients -
+  const numberOfRecipients = sequelize.query(numberOfRecipSql, {
+    type: QueryTypes.SELECT,
+    transaction,
+  });
+
+  // - Reports with Resources Pct -
+  const pctOfResourcesSql = `
+  SELECT
+    count(DISTINCT "activityReportId")::decimal AS "reportsWithResourcesCount",
+    ${totalReportCount}::decimal AS "totalReportsCount",
+    CASE WHEN ${totalReportCount} = 0 THEN
+      0
+    ELSE
+      (round(count(DISTINCT "activityReportId")::decimal / ${totalReportCount}::decimal, 4) * 100)::decimal
+    END AS "resourcesPct"
+  FROM ${tblNames.createdAroResourcesTempTableName};
+  `;
+  const pctOfReportsWithResources = sequelize.query(pctOfResourcesSql, {
+    type: QueryTypes.SELECT,
+    transaction,
+  });
+
+  // - Number of Reports with ECLKC Resources Pct -
+  const pctOfECKLKCResources = sequelize.query(`
+    WITH eclkc AS (
+      SELECT
+    COUNT(DISTINCT url) AS "eclkcCount"
+    FROM  ${tblNames.createdFlatResourceTempTableName}
+    WHERE url ilike '%eclkc.ohs.acf.hhs.gov%'
+    ), allres AS (
+    SELECT
+    COUNT(DISTINCT url) AS "allCount"
+    FROM ${tblNames.createdFlatResourceTempTableName}
+    )
+    SELECT
+      e."eclkcCount",
+    r."allCount",
+      CASE WHEN
+    r."allCount" = 0
+    THEN 0
+    ELSE round(e."eclkcCount" / r."allCount"::decimal * 100,4)
+    END AS "eclkcPct"
+    FROM eclkc e
+    JOIN allres r
+      ON 1=1;
+  `, {
+    type: QueryTypes.SELECT,
+    transaction,
+  });
+
+  // 5.) Date Headers table.
+  const dateHeaders = sequelize.query(`
```

Comment:
please add the /* sql */ at the start of the query string to allow for syntax highlighting to make the code easier to read.